MattWang05.md

```diff
@@ -0,0 +1,56 @@
+---
+name: "@MattWang05"
```

Comment:
```suggestion
name: ["@MattWang05", "@choy1015"]
```

Reply:
score) We'll determine how much the accelerometer accelerated and adjusted the "power" based on that. We'll adjust the scaling as we test it out and the score will be displayed on an LCD screen that comes with the kit that we included in our budget.

music player 1) Yes, we plan on using one of ours
music player 2) We plan on starting small with the number of RFID stickers and will buy more later if needed
music player 3) It's to hide the raspberry pi 4 and the other materials from sight. It's to make the final result look cleaner.